public/styles/site/_custom.scss

```diff
@@ -163,6 +163,14 @@ small {
   border: 1px solid $primary-tan;
 }
 
+.col-xs-12 {
```

Comment:
You are overriding a framework style here rather than adding additional behavior by creating a new class. Fix this @mattphoto